chrome-extension/package.json

```
@@ -31,6 +31,7 @@
     "ts-loader": "^9.5.1",
     "deepmerge": "^4.3.1",
     "cross-env": "^7.0.3",
-    "@types/node": "^20.16.3"
+    "@types/node": "^20.16.5",
+    "vite": "^5.4.3"
```

It's already installed as root, so I guess I can uninstall it!

```suggestion
    "@types/node": "^20.16.5"
```

---

Yeah, you're right.
I want to do it, but i've forgot, because i have problem with this lack of `turbo ready` for `dev` and it doesn't working.
It working perfect without it, because it's redundancy 😄